packages/surveycto/src/Adaptor.js

```diff
@@ -174,6 +190,47 @@ export function cursor(value, options) {
   return commonCursor(value, opts);
 }
 
+/**
+ * The function `jsonToCSVBuffer` takes in an array of objects and creates a CSV Buffer
+ * @public
+ * @example
+ * jsonToCSVBuffer([
+ *  {
+ *     lastName: 'Rothfuss', 
+ *     firstName: 'Patrick', 
+ *     book: 'The Name of the Wind'
+ *   },
+ *  {
+ *     lastName: 'Sanderson',
+ *     firstName: 'Brandon', 
+ *     book: 'The Way of Kings'
+ *   },
+ *  {
+ *     lastName: 'Martin',
+ *     firstName: 'George', 
+ *     book: 'A Game of Thrones'
+ *   },
+ * ])
+ * @param {*} rows An array of JSON objects.
+ * @returns {Operation}
+ */
+export function jsonToCSVBuffer(rows) {
```

Comment:
Just for the record, I'm a little worried about writing the buffer here to the state object.

I've run a test with the CLI locally and it seems to be OK, it just gives you weird output.

I would be a bit more comfortable if this was a public util function, like`util.jsonToCSVBuffer()`, which just returns a value. No state stuff.

But I'm not going to block this PR on it.